styles/colors-gallium-day.less

```diff
@@ -215,14 +215,16 @@
 
 // SliderButton
 // ---------------------------------------
-@agate-sliderbutton-color:          @agate-accent;
+@agate-sliderbutton-color:          @agate-white;
```

Comment:
Maybe a darker `@agate-sliderbutton-color` would make the text more legible for Gallium Day? Right now it's a bit hard to read white text on light gray.

<img width="649" alt="SliderButton" src="https://user-images.githubusercontent.com/8940009/71855952-4db31b80-3097-11ea-9b86-765b5f23817f.png">

Reply:
I updated these to be `@agate-text-color` in both skins, which seems more appropriate. Declaring plain white, like I did before, seemed wrong, while this seems more in-line with the skins' core approach.